src/app/pages/signing/signing.component.html

```diff
@@ -1,61 +1,73 @@
-<div class="container py-5 center-children vertical-center">
-  <div class="row">
-    <h2>Sending Message to friend.</h2>
+<div class="container py-5 d-flex vertical-center">
+  <div class="row" id="title-div">
+    <label for="signing-box">
+      <h2 id="signing-header">Type your message to [friend name] below.</h2>
+    </label>
   </div>
-  <div class="row">
-    <textarea rows="8" class="form-control" id="signing-box" autofocus [(ngModel)]='signingBoxText'></textarea>
+  <div class="d-flex justify-content-center col-12">
+    <div class="row">
+      <textarea #inputarea rows="6" class="form-control" id="signing-box" autofocus [(ngModel)]='signingBoxText'
```

Comment:
You should rename `inputarea` to `inputArea`

Reply:
There are also other instances of 'inputarea' in this file just so you know.